src/common/pa_converters.c

```diff
@@ -329,24 +330,62 @@ static const double const_1_div_2147483648_ = 1.0 / 2147483648.0; /* 32 bit mult
 
 /* -------------------------------------------------------------------------- */
 
+static inline int SetRoundingMode()
+{
+    int prev = fegetround();
+
+    if (fesetround(FE_TONEAREST) != 0)
+        prev = -1;
+
+    return prev;
+}
+
+/* -------------------------------------------------------------------------- */
+
+static inline void ResetRoundingMode(int prev)
```

Comment:
This could be called SetRoundingMode(mode).

Reply:
We use this function to reset the mode. But the function itself just sets the mode. 
The function should be named for what it does not how we use it.

Reply:
I missed your previous comment regarding it, updated PR to reflect the proposal.